### PR TITLE
AFT. Payment Contexts. Account Holder. Other Updates

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
## Changes
- Update AFT properties and Fix Tests
- Update payment contexts
- Fix int nullables
- Tax rate update
- Update account holder and payment responses
- Add network token destination
- Update Payment Method Configuration Applepay and Googlepa
- Update Disputes

### CODE REVIEW

File: src/CheckoutSdk/Payments/PaymentDestinationType.cs [hunk 1]
```diff
@@ -8,5 +8,6 @@ public enum PaymentDestinationType
         [EnumMember(Value = "card")] Card,
         [EnumMember(Value = "id")] Id,
         [EnumMember(Value = "token")] Token,
+        [EnumMember(Value = "network_token")] Networktoken
```
Comment (robert-goheen-cko) [inline src/CheckoutSdk/Payments/PaymentDestinationType.cs:11]: Shouldn't this be `NetworkToken` ?

Comment (armando-rodriguez-cko) [reply src/CheckoutSdk/Payments/PaymentDestinationType.cs:11]: 👍🏼 💯

File: src/CheckoutSdk/Payments/ProductType.cs [hunk 2]
```diff
@@ -4,9 +4,40 @@ namespace Checkout.Payments
...
+        [EnumMember(Value = "Official Account")]
+        OfficialAccount,
+
+        [EnumMember(Value = "Mini Program")]
+        MiniProgram,
+
+        [EnumMember(Value = "pay_in_full")]
+        PayInFull,
+
+        [EnumMember(Value = "pay_by_instalment")]
```
Comment (robert-goheen-cko) [inline src/CheckoutSdk/Payments/ProductType.cs:22]: Why are the new enum vals a different format than the older ones?

Comment (armando-rodriguez-cko) [reply src/CheckoutSdk/Payments/ProductType.cs:22]: To read easier: https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=0/processing/product_type&t=request

**Pull request merged**